bundle.yaml

```diff
@@ -37773,6 +37773,8 @@ spec:
       securityContext:
         runAsNonRoot: true
         runAsUser: 65534
+        seccompProfile:
+          type: RuntimeDefault
```

Comment:
Thanks for the patch! `bundle.yaml` is a generated file (see makefile target `bundle.yaml`). Below may be what you are looking for, followed by make generate. 

```
diff --git a/jsonnet/prometheus-operator/prometheus-operator.libsonnet b/jsonnet/prometheus-operator/prometheus-operator.libsonnet
index 7e2688e06..49fa5b37c 100644
--- a/jsonnet/prometheus-operator/prometheus-operator.libsonnet
+++ b/jsonnet/prometheus-operator/prometheus-operator.libsonnet
@@ -33,7 +33,7 @@ function(params) {
                                                  if po.config.enableAlertmanagerConfigV1beta1 then
                                                    (import 'alertmanagerconfigs-v1beta1-crd.libsonnet')
                                                  else {},
-  '0prometheusagentCustomResourceDefinition': import 'prometheusagents-crd.json',
+  '0prometheusagentCustomResourceDefinition': import 'prometheusagents-crd.json',
   '0prometheusCustomResourceDefinition': import 'prometheuses-crd.json',
   '0servicemonitorCustomResourceDefinition': import 'servicemonitors-crd.json',
   '0podmonitorCustomResourceDefinition': import 'podmonitors-crd.json',
@@ -178,6 +178,7 @@ function(params) {
             securityContext: {
               runAsNonRoot: true,
               runAsUser: 65534,
+              seccomProfile: 'RuntimeDefault',
             },
             serviceAccountName: po.config.name,
             automountServiceAccountToken: true,
```

Reply:
I think I've made the requested changes, the libsonnet does make a lot more sense.